backend/src/procedure/js_scripts.rs

```diff
@@ -120,11 +121,16 @@ mod js_runtime {
 
     use crate::s9pk::manifest::PackageId;
     use crate::util::Version;
-    use crate::volume::{script_dir, Volumes};
+    use crate::volume::script_dir;
+    use crate::volume::Volumes;
 
     use super::super::ProcedureName;
     use super::{JsCode, JsError};
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
These probably aren't compatible with eachother. So the snapshot is likely x86_64 only.

Reply:
K